Doc/library/math.rst

```diff
@@ -374,6 +376,24 @@ Floating point manipulation functions
    .. versionadded:: 3.2
 
 
+.. function:: isnormal(x)
+
+   Return ``True`` if *x* is a normal floating-point number, that is a finite
+   nonzero number that is not a subnormal (see :func:`issubnormal`).
+   Return ``False`` otherwise.
```

Comment:
```suggestion
   Return ``True`` if *x* is a normal number, that is a finite
   nonzero number that is not a subnormal (see :func:`issubnormal`).
   Return ``False`` otherwise.
```